flink-core/src/main/java/org/apache/flink/configuration/NettyShuffleEnvironmentOptions.java

```diff
@@ -491,6 +491,36 @@ public class NettyShuffleEnvironmentOptions {
                                     + " based on the platform. Note that the \"epoll\" mode can get better performance, less GC and have more advanced features which are"
                                     + " only available on modern Linux.");
 
+    @Documentation.Section(Documentation.Sections.ALL_TASK_MANAGER_NETWORK)
+    public static final ConfigOption<Integer> CLIENT_TCP_KEEP_IDLE_SECONDS =
+            key("taskmanager.network.netty.client.tcp.keepIdleSec")
+                    .intType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "The Netty client tcp keepalive idle time in seconds. "
+                                    + "Note: This doesn't take into account if using netty transport type of nio and an older version of jdk 8, "
```

Comment:
```suggestion
                                    + "Note: This will not take effect when using netty transport type of nio with an older version of JDK 8, "
```

Reply:
Same as below.